{"name": "输入错误密码登录失败", "status": "passed", "attachments": [{"name": "stdout", "source": "ceb4aa84-de50-46cf-9a0b-7159d7d7571a-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '5', 'casename': '输入错误密码登录失败', 'mode': '登录', 'data': 'userAccount=979172251@qq.com\\npassword=a1234567', 'assert_way': 'class name=login-form-forgot', 'result': '\\n              Forgot Password?\\n            '}"}], "start": 1650603635613, "stop": 1650603640173, "uuid": "87e53f25-813e-4041-b2da-ce9cd20e9fa7", "historyId": "104e3d5feb09afaca2b374c9a621f082", "testCaseId": "f42af1ba93a36a6b78bc4dd2bc5b083c", "fullName": "test_main.Test_Main#test_login", "labels": [{"name": "feature", "value": "登录"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "9384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}